dataflow-website/recipes/scaling/scdf-alert-webhook/README.md

```diff
@@ -0,0 +1,20 @@
+
+# Auto-scaling adapter for Streaming Data Pipelines
+
+The `AlertWebHookApplication` is a [Prometheus Alertmanager Webhook Receiver](https://github.com/prometheus/alertmanager) 
```

Comment:
Is "Webhook Receiver" a known terminology in Prometheus?

Reply:
Yes, I use to call it alert handler initially but renamed to wehook receiver to match the prometheus terminology. There is a link there as well.